cs-config/cs_config/functions.py

```diff
@@ -163,9 +163,7 @@ def run_model(meta_param_dict, adjustment):
     # Solve baseline model
     base_spec = {
         **{'start_year': meta_param_dict['year'],
-        'debt_ratio_ss': 2.0, 'r_gov_scale': 1.0, 'r_gov_shift': 0.02,
-        'zeta_D': [0.4], 'zeta_K': [0.1], 'initial_debt_ratio': 0.78,
-        'initial_foreign_debt_ratio': 0.4, 'tax_func_type': 'linear',
+        'tax_func_type': 'linear',
```

Comment:
@jdebacker. Why this change in the baseline specification to `'tax_func_type': 'linear',`?

Reply:
This is for C/S so that things run faster.  Linear tax functions take about 15 minutes, whereas the `DEP` functions take a couple hours to estimate.   We are also doing `age_dependent=False` to save computational time for now.

Reply:
OK